### DISCUSSION TITLE
North Ecliptic Pole as an option as a Solar System Observer

### DISCUSSION THREAD

**Discussion by LeviDavis65** [Stellarium/stellarium#3696] [Ideas]
Hello Stellarium Team and Community,

I hope this message finds you well. I'm a passionate user of Stellarium, fascinated by the unique perspectives it offers on our universe. Through Stellarium, we can explore the cosmos from various points within our Solar System, gaining insights and understanding that ground-based observations cannot provide. To enhance this exploratory experience, I propose the addition of two significant celestial observation points: the North Ecliptic Pole (NEP) and the South Ecliptic Pole (SEP).

Why the NEP and SEP?

The Ecliptic Poles offer a perpendicular view of the ecliptic plane, providing a unique vantage point for observing the Solar System's dynamics. From these points, users can visualize the Solar System's layout and motion in a way that is not possible from within the ecliptic plane itself. Such a perspective is invaluable for educational purposes, helping to explain concepts like the ecliptic, celestial coordinates, and the apparent motion of Solar System objects.

Educational Value:

Adding the NEP and SEP as observation points could greatly benefit educators and students by providing a practical tool to visualize and understand celestial mechanics. It would offer an intuitive way to grasp the alignment of the planets with the ecliptic plane and their relative positions and motions through the sky.

Astronomical Research and Hobbyist Exploration:

For researchers and astronomy enthusiasts, these points offer a novel perspective for planning observations, understanding celestial events, and exploring theoretical scenarios. It opens up new possibilities for amateur and professional astronomers alike to simulate and visualize celestial phenomena from an out-of-the-ecliptic viewpoint.

Implementation Consideration:

Understanding the complexity of implementing new features, I believe this addition could leverage the existing framework for Solar System Observer points. The NEP and SEP, defined by their celestial coordinates (Right Ascension and Declination), could be integrated as fixed points of interest from which the simulation can be observed.

Conclusion:

I strongly believe that including the NEP and SEP as optional observation points would enrich Stellarium's educational and exploratory capabilities. It would not only expand the software's utility but also inspire curiosity and further understanding of our place in the universe.

I appreciate your consideration of this proposal and am keen to hear your thoughts and feedback. Thank you for your ongoing dedication to making Stellarium a window to the cosmos for us all.

Best regards,

Levi

**Comment by gzotti** [comment]
Like this?
![grafik](https://github.com/Stellarium/stellarium/assets/18099873/aab08994-1aae-4593-b9f7-fa9c55910842)

Just use Solar System Observer and move up (Alt-CursorUp).

**Reply by candylandchampion** [reply]
I'm using v.24.4 on mac.  Fn-Option-right arrow is not working to put SS Observer at the NEP.  Do you know the keyboard shortcut for mac?
